lib/pf/radius.pm

```diff
@@ -938,10 +941,20 @@ Handle NTLM caching if necessary
 
 sub handleNtlmCaching {
     my ($self, $radius_request) = @_;
+    my $logger = get_logger;
     my $domain = $radius_request->{"PacketFence-Domain"};
+    my $usedNtHash = $radius_request->{"PacketFence-NTCacheHash"};
+
     if($domain && isenabled($ConfigDomain{$domain}{ntlm_cache}) && isenabled($ConfigDomain{$domain}{ntlm_cache_on_connection})) {
-        my $client = pf::api::queue->new(queue => "general");
-        $client->notify("cache_user_ntlm", $domain, $radius_request->{"Stripped-User-Name"});
+        my $CHI_CACHE = pf::CHI->new( namespace => 'ntlm_cache_username_lookup' );
```

Comment:
Create a sub in pf::domain::ntlm_cache to get the cache and use it here and in pf::domain::ntlm_cache as well